common/djangoapps/student/management/tests/test_populate_user_data_on_braze.py

```diff
@@ -0,0 +1,92 @@
+"""
+Unittests for populate_marketing_opt_in_user_attribute management command.
+"""
+from unittest.mock import patch, MagicMock
+
+from braze.exceptions import BrazeClientError
+from django.core.management import call_command
+from django.test import TestCase
+from testfixtures import LogCapture
+
+from common.djangoapps.student.tests.factories import UserFactory, UserAttributeFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+MARKETING_EMAILS_OPT_IN = 'is_marketable'
+LOGGER_NAME = 'common.djangoapps.student.management.commands.populate_user_data_on_braze'
+
+
+@skip_unless_lms
+class TestPopulateUserDataOnBraze(TestCase):
+    """
+    Tests for PopulateUserDataOnBraze management command.
+    """
+
+    def setUp(self):
+        super().setUp()
+        for index in range(15):
+            user = UserFactory()
+            UserAttributeFactory(user=user, name=MARKETING_EMAILS_OPT_IN, value=(index % 2 == 0))
```

Comment:
The setup function executes before each test run in the class, move this into `setupClass` if required only once.